.github/workflows/ci.yml

```
@@ -0,0 +1,68 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v4
+
+      - name: Test Local Action
+        id: test-action
+        uses: ./
+        with:
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

actionlint (1.7.4)

input "GH_TOKEN" is not defined in action "MySQL migration" defined at "./". available inputs are "mysql-database", "mysql-host", "mysql-user", "operation"

(action)